src/python/pants/engine/native.py

```diff
@@ -257,33 +257,45 @@ def get_build_cflags():
 
 def bootstrap_c_source(output_dir, module_name=NATIVE_ENGINE_MODULE):
   """Bootstrap an external CFFI C source file."""
+
+  tempdir = safe_mkdtemp()
```

Comment:
how about using `pants.util.contextutil.temporary_dir()` or one of the other utils in there as a contextmgr here for automatic tmp dir cleanup?

Reply:
Done